daffodil-test/src/test/scala/org/apache/daffodil/section00/general/TestGeneral.scala

```diff
@@ -97,7 +97,7 @@ class TestGeneral {
 
   @Test def test_1530_scala_version() {
     val versionStr = scala.util.Properties.versionNumberString
-    assertTrue("The scala version must begin with 2.11. The current version is: " + versionStr, versionStr.startsWith("2.11."))
+    assertTrue("The scala version must begin with 2.11 or 2.12. The current version is: " + versionStr, versionStr.startsWith("2.11.") || versionStr.startsWith("2.12."))
   }
```

Comment:
I'm not even sure this test makes is necessary. Having to update this test everytime we bump the scala version seems unnecessary. As long as all the tests pass under a certain version of Scala, we should be happy. I'd say we should just remove this test.